.github/workflows/kind-e2e.yaml

```
@@ -118,6 +119,11 @@ jobs:
         - ingress: istio
           namespace-resources: virtualservices
 
+        - ingress: istio-tls
+          ingress-class: istio
+          namespace-resources: virtualservices
```

```suggestion
          namespace-resources: virtualservices,destinationrules
```

We would like to get the `destinationrules` for debug when system TLS is enabled?

---

Definitely 👍